luigi/file.py

```diff
@@ -81,14 +81,16 @@ def __init__(self, path=None, format=None, is_tmp=False):
         self.format = format
         self.is_tmp = is_tmp
 
+    def mkdirs(self):
```

Comment:
Up to you. I talk sed and awk fluently so mkdirs is a least terrible thing I do in my work :D